src/lemmy_client.rs

```diff
@@ -126,51 +131,85 @@ cfg_if! {
             }
         }
 
-        #[async_trait(?Send)]
-        impl private_trait::LemmyClient for awc::Client {
+        impl private_trait::PrivateFetch for LemmyClient {
             async fn make_request<Response, Form, Request>(
                 &self,
                 method: HttpType,
                 path: &str,
                 req: Request,
             ) -> LemmyAppResult<Response>
             where
-                Response: Serializable + for<'de> Deserialize<'de>,
-                Form: Serialize,
-                Request: Into<LemmyRequest<Form>>
+                Response: Serializable + for<'de> Deserialize<'de> + 'static,
+                Form: Serialize + core::clone::Clone + 'static + core::fmt::Debug,
```

Comment:
`Clone` is in the prelude. Unless you're writing macros, you really don't need to fully qualify it everywhere.

Reply:
i thought that should be the case but decided to copy it for consistency. i wonder how little of the prelude you need to be valid? just what you're using like a `use`?